sentences/it/fan_HassTurnOff.yaml

```diff
@@ -6,5 +6,5 @@ intents:
           - "<turn_off> [(tutt(i | e))] [<the>] <fan> [(<of> | <in>)] <area>"
           - "<turn_off> [<in>] <area> [(tutt(i | e))] [<the>] <fan>"
```

Comment:
Please, follow new rules as in new version of hassil #830.
There sould not be spaces around `|`